src/js/controller/controller.js

```diff
@@ -337,12 +338,26 @@ Object.assign(Controller.prototype, {
             if (model.get('playOnViewable')) {
                 if (viewable) {
                     _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
```

Comment:
Do we really want to keep this mobile behavior?

Reply:
This is how we pause and resume on mobile web when not viewable.

Reply:
Sure, but why not just enable autopause on mobile by default? Why do we need more code to do the same thing?